File: yangxj96-serve/yangxj96-serve-platform/src/main/java/io/github/yangxj96/server/platform/Application.java
package io.github.yangxj96.server.platform;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * 平台内容的主启动类
 */
@MapperScan("io.github.yangxj96.server.platform.mapper")
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: yangxj96-serve/yangxj96-serve-platform/src/main/java/io/github/yangxj96/server/platform/Application.kt
package io.github.yangxj96.server.platform

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * 平台内容的主启动类
 */
@MapperScan("io.github.yangxj96.server.platform.mapper")
@EnableDiscoveryClient
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}